### PR TITLE
[fix](nereids) Fix load failed where not set database in session

### PR DESCRIPTION
**Pull request by zxealous**
## Proposed changes

Issue Number: close #xxx

load failed where not set database in session

```
LOAD LABEL test_db.label_111111 ( DATA INFILE("hdfs://hdfs01:9000/user/") INTO TABLE `test_load_tb`) WITH BROKER "broker" ( "username" = "user",  "password" = "");

ERROR 1105 (HY000): errCode = 2, detailMessage = Current database is not set.
```

use label's database if not set database in session

### CODE REVIEW

**Comment by zxealous** [comment]
run buildall

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4 [hunk 1]
```diff
@@ -920,8 +920,8 @@ dataDesc
         (deleteOn=deleteOnClause)?
         (sequenceColumn=sequenceColClause)?
         (propertyClause)?
-    | ((WITH)? mergeType)? DATA FROM TABLE tableName=multipartIdentifier
-        INTO TABLE tableName=multipartIdentifier
+    | ((WITH)? mergeType)? DATA FROM TABLE tableName=identifier
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:923]:
```suggestion
    | ((WITH)? mergeType)? DATA FROM TABLE sourceTableName=identifier
```

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4 [hunk 2]
```diff
@@ -920,8 +920,8 @@ dataDesc
...
+        INTO TABLE tableName=identifier
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:924]:
```suggestion
        INTO TABLE targetTableName=identifier
```

File: fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4 [hunk 3]
```diff
@@ -906,7 +906,7 @@ identityOrFunction
 
 dataDesc
     : ((WITH)? mergeType)? DATA INFILE LEFT_PAREN filePaths+=STRING_LITERAL (COMMA filePath+=STRING_LITERAL)* RIGHT_PAREN
-        INTO TABLE tableName=multipartIdentifier
+        INTO TABLE tableName=identifier
```
Comment (morrySnow) [inline fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4:909]:
```suggestion
        INTO TABLE targetTableName=identifier
```

**Comment by zxealous** [comment]
run buildall

**Comment by zxealous** [comment]
run buildall

**Comment by zxealous** [comment]
run buildall

**Review by adonis0147 — APPROVED** [review]

**Review by morrySnow — APPROVED** [review]

**Pull request merged**